presto-native-execution/presto_cpp/main/types/PrestoToVeloxConnector.h

```diff
@@ -32,14 +32,14 @@ void registerPrestoToVeloxConnector(
 void unregisterPrestoToVeloxConnector(const std::string& connectorName);
 
 const PrestoToVeloxConnector& getPrestoToVeloxConnector(
-    const std::string& connectorId);
+    const std::string& connectorName);
 
 class PrestoToVeloxConnector {
  public:
   virtual ~PrestoToVeloxConnector() = default;
 
-  [[nodiscard]] const std::string& connectorId() const {
-    return connectorId_;
+  [[nodiscard]] std::string connectorName() const {
```

Comment:
Can we return this by reference?
The caller can decide to make a copy or to continue to keep a reference.

Reply:
Right. I was debating this but decided that I should change to reduce possible mistakes later. I can put it back.

Reply:
I discussed this with the author in the original PR. A reference is always returned in general for other objects. So we decided to leave it at that.